### ISSUE TITLE
Are there any plans for this app in the near future?

### ISSUE THREAD

**Issue by MeiKatz** [nextcloud/social#1518]
To be honest, especially after Twitter went mad and Mastodon receives a lot of new users in the last days, an app like this would be great to have. So it's a pity that there were no progress in the last two years.

**Are there any plans for this app in the near future?**

And what are the reasons why this app has not seen any development in the last months/years? Do you need a new maintainer? Do you need new contributors?

Please let the community know how we can support you and we can bring this app back and to new glory!

**Comment by ArtificialOwl** [comment]
we're currently working on a future release for NC25

**Comment by MeiKatz** [comment]
That are great news! I am really excited to see where the road goes and like to use it for my organisation. Maybe within the greater outer world of Mastodon?

**Comment by f4grx** [comment]
Adding my interest.

Is there any way to help for general users? Testing? Translation?

**Comment by ArtificialOwl** [comment]
Right now, we have already listed a lot of fixes/improvement for next beta release that should be available for December. You should wait for this version and start trying it out.

**Comment by f4grx** [comment]
Hi, Thanks for the reply, we will wait. It's good to know that the project is alive and well!

**Comment by ssaavedra** [comment]
Should we keep this issue open for such notifications? I'd like a place to be added to be notified about such an update :)

**Comment by biglari** [comment]
There was a new version [v0.4.2](https://github.com/nextcloud/social/releases/tag/v0.4.2) on the 7th.

**Comment by c0urier** [comment]
> Right now, we have already listed a lot of fixes/improvement for next beta release that should be available for December. You should wait for this version and start trying it out.

December or January?

**Comment by soehlschlaeger** [comment]
At this moment, the beta is running with NC25 on my Webserver. But since I have testing it out the website is freezing if I'm loading the social startpage. There no function left and the only this I could do is to close the window. Anybody out there with this error?

**Comment by gbraad** [comment]
> At this moment, the beta is running with NC25 on my Webserver. 

No beta was released to my knowledge.

**Comment by soehlschlaeger** [comment]
Thanks. I will try it. FYI Here is the overview for the beta I've used. https://help.nextcloud.com/t/social-beta-released-11-2-2022/148924

**Comment by gbraad** [comment]
@soehlschlaeger the current buildscript has some missing dependencies. Filed in the other issue. Will check the beta too,

**Comment by gbraad** [comment]
the beta works for me. Although it does not setup `.well-known` for webfinger automagically. trying to figure this out, as at the moment it can not receive remote messages (or send them)

**Comment by soehlschlaeger** [comment]
After the install it worked completely for 1 day. After I play arround it freeze at the homepage. 

Thank for testing.

**Comment by gbraad** [comment]
what do you mean with 'freeze'. Any messages in the developer tools of the browser or the `nextcloud.log` in `./data`?

**Comment by soehlschlaeger** [comment]
https://user-images.githubusercontent.com/58472193/210506200-f33ea720-67a4-4158-a291-65d5354cf5f4.MOV

The website is crashing on all devices. The only thing you see is the rolling ball. No keyboard, finger or mouse inputs possible and I have to close the tab.

**Comment by soehlschlaeger** [comment]
I will check the logfiles later.

**Comment by gbraad** [comment]
> rolling ball

You mean the macOS beach ball? if this freezes, perhaps it sounds like the webpages is having an issue with javascript or redirects.

**Comment by soehlschlaeger** [comment]
Not the Beachball. This one https://user-images.githubusercontent.com/58472193/210512208-1d88eb4a-1e6d-46cb-bbf3-42a2b861d54e.MOV

**Comment by gbraad** [comment]
That is called a spinner.
Check your browser console for more information.

**Comment by Thatoo** [comment]
> we're currently working on a future release for NC25

Will it be the 0.4 alpha with all what is in the milestone including the [data Import/Export](https://github.com/nextcloud/social/issues/118)?

**Comment by Thatoo** [comment]
The 0.4 alpha milestone is Past due by over 2 years, Last updated over 1 year ago and is 40 % completed.
It look like it doesn't get the attention it deserve nor announcements [from this blog post](https://nextcloud.com/blog/nextcloud-introduces-social-features-joins-the-fediverse/) are respected finally.
Sadly.

With a little bit of dev, could it not become the "linkedin replacement" in the fediverse...?

**Comment by MeiKatz** [comment]
> The 0.4 alpha milestone is Past due by over 2 years, Last updated over 1 year ago and is 40 % completed.
> It look like it doesn't get the attention it deserve nor announcements [from this blog post](https://nextcloud.com/blog/nextcloud-introduces-social-features-joins-the-fediverse/) are respected finally.
> Sadly.
> 
> With a little bit of dev, could it not become the "linkedin replacement" in the fediverse...?

I guess it would be easy to create a fork or take over the maintenance of this repository. Even if there was no greater goal for it in the near future. Any progress is better than none.

**Comment by soehlschlaeger** [comment]
Hey. This is a living Project with a lot of updates on the last half year. 

I'm only an user, but the beta ist running okay, but not sensationally.

**Comment by MeiKatz** [comment]
> Hey. This is a living Project with a lot of updates on the last half year. 
> 
> I'm only an user, but the beta ist running okay, but not sensationally. 

Yes, I checked the last commits. Maybe the best would be adding pull requests by myself and helping the guys behind the repository.

**Comment by Thatoo** [comment]
ok is ok, sensationally would be nice :smile: .
Nextcloud and all the main apps are sensational but social is ok.

**Comment by karlitschek** [comment]
Closing this as answered. Development has continued: the app is at 0.11.63 with moderation, retention, blocking and muting, reporting, polls, quote posts, hashtag following, and a substantial part of the Mastodon client API.

For specific gaps, individual issues are easier to track than this thread.
